examples/plot_interpolate_bad_channels.py

```diff
@@ -51,7 +49,7 @@
 evoked_before.plot(exclude=[])
 
 # compute interpolation (also works with Raw and Epochs objects)
-evoked_after = evoked_before.copy().interpolate_bads_eeg()
+evoked_after = evoked_before.interpolate_bads()
 
-# plot interpolated (prevsious bads)
+# plot interpolated (previous bads)
 evoked_after.plot(exclude=[])
```

Comment:
okay, but can we somehow specify that this happens in the info? `reset_bads` somehow tells me that this is something that happens to the data.